stable/wordpress/values.yaml

```diff
@@ -39,6 +39,28 @@ wordpressLastName: LastName
 ##
 wordpressBlogName: User's Blog!
 
+## Database name to create
+## Sets WORDPRESS_DATABASE_NAME and MYSQL_CLIENT_CREATE_DATABASE_NAME
+## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
+databaseName: bitnami_wordpress
+
+## Database user to create
+## Sets WORDPRESS_DATABASE_USER and MYSQL_CLIENT_CREATE_DATABASE_USER 
+## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
+databaseUser: bn_wordpress
+
+## Database password for the new user
+## Sets WORDPRESS_DATABASE_PASSWORD  and  MYSQL_CLIENT_CREATE_DATABASE_PASSWORD 
+## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
+## databasePassword: 
+
+## Set to `yes` to allow the container to be started with blank passwords
+## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
+## Defaults to no
+## allowEmptyPassword: 
```

Comment:
we generally add the default value to the values.yaml file. I'd suggest un-commenting changing this to

```
allowEmptyPassword: "no"
```

or if you think https://github.com/kubernetes/charts/pull/773/files#r105334999 makes sense, then you should be using

```
allowEmptyPassword: false
```